README.md

```diff
@@ -1,146 +1,79 @@
 
-## What is it? ##
+All the implementation uses the HTTP protocol, since most of Stardog functionality is available using this protocol. For more information, go to the Stardog's [HTTP Programming](http://www.stardog.com/docs/#_network_programming) documentation.
 
-This framework wraps all the functionality of a client for the Stardog DBMS, and provides access to a full set of functions such as executing SPARQL Queries, administration tasks on Stardog, and the use of the Reasoning API.
-
-All the implementation uses the HTTP protocol, since most of Stardog functionality is available using this protocol. For more information, go to the Stardog's [HTTP Programming](http://docs.stardog.com/http/) documentation.
-
-The framework is currently supported for node.js and the browser, including test cases for both environments.
-You'll also need [npm](https://npmjs.org) and [bower](http://bower.io) to run the test cases and install the dependencies in node.js & the browser respectively.
+This is a universal library and as such can be used in both the browser and Node.
 
 ## Installation
 
-To install stardog.js locally from the npm registry simply execute:
-
-#### In node.js
-
-    npm install stardog
-
-That will fetch the latest version of stardog.js in the npm registry, [more details](https://npmjs.org/package/stardog).
+To install stardog.js run:
 
-#### In the browser (client library using bower)
-
-    bower install stardog
-
-That will fetch the latest version of stardog.js in the bower registry, [more details](http://bower.io/search/?q=stardog).
+```bash
+npm install stardog
+```
 
-## Development ##
+## Development
 
 To get started, just clone the project. You'll need a local copy of Stardog to be able to run the tests. For more information on starting the Stardog DB service and how it works, go to [Stardog's documentation](http://docs.stardog.com), where you'll find everything you need to get up and running with Stardog.
 
 Go to [http://stardog.com](http://stardog.com), download and install the database and load the data provided in `data/` using the script in the repository.
 
 1. Start the Stardog server
-
-    * Stardog > `2.1.1`
-
-            $ stardog-admin server start
-
-    * Stardog <= `2.1.1`
-
-            $ stardog-admin server start --port 5823
-
+```bash
+stardog-admin server start
+```
 2. Install `stardog.js` dependencies:
-
-        $ npm install
-        $ bower install
-
-This will install all the dependencies using npm (for node.js) and bower (for browser), once this is done, run the test cases.
+```bash
+npm install
+```
 
 ### Running Tests
 
-Run all the test cases in `test/spec`, all test cases must pass. Having the Stardog server running, execute the following commands:
-
-* In the browser
-
-    1. Load the test data using the provided script:
+Run all the test cases in `test/spec`, all test cases must pass. With the Stardog server running, execute the following commands:
```

Comment:
"In order to contribute changes, all test cases must pass. With the Stardog server running, execute the following:"?